get-started/code-experiments/README.md

```diff
@@ -0,0 +1,260 @@
+## DVC Get Started for Experiments
+
+This repository contains a project that trains a classifier for [Fashion
+MNIST][fmnist] dataset in Tensorflow. 
+
+[fmnist]: https://github.com/zalandoresearch/fashion-mnist
```

Comment:
> don't use [links][like this] unless you re-use them

On this, seems like a stylistic choice TBH (up to eahc person for now). But I would agree it's best to embed them most times unless

a) the URLs are super long
b) they are not very short and repeat (esp if more than once)
c) they contain a URL anchor e.g. `/some/path#specific-section-title`

What do you guys think?

Reply:
I don't think this repo uses "raw" markdown - it's actually "code" markdown so this makes more sense to me:

- keep `[with the link text](URL)`
- unless the URL is re-used, in which case avoid `[duplication & typos][URL]` with `[aliases][URL]`

I think we should be using language features to our advantage rather than 1. because the features exist or 2. based on style preferences

Reply:
For me, the difference is this:

![levinas-210602-114952](https://user-images.githubusercontent.com/476310/120450219-ee932c80-c398-11eb-86df-1a566f3dd0e6.png)
![levinas-210602-114918](https://user-images.githubusercontent.com/476310/120450243-f521a400-c398-11eb-9d86-4326c6a95bcf.png)

I admit the [reference links] are rare in MD documents, but seeing paragraphs broken by inline links makes my OCD side suffer. :)) The broken links are a different matter; you can have broken links in (inline) links as well. 

I changed the links in iterative/dvc.org#2497 because the link checker wasn't aware of them. Here, however, we should consider people will be reading the raw `README.md` file in their editor as well. In that case having reference links is cleaner IMO.

Reply:
For me, the difference is this:

![levinas-210602-114952](https://user-images.githubusercontent.com/476310/120451799-6746b880-c39a-11eb-839a-7817e1956662.png)
![levinas-210602-114918](https://user-images.githubusercontent.com/476310/120451804-6877e580-c39a-11eb-9cb0-153cf4528c08.png)


I admit the [reference links] are rare in MD documents, but seeing paragraphs broken by inline links makes my OCD side suffer. :)) The broken links are a different matter; you can have broken links in (inline) kind as well. 

I changed the links in iterative/dvc.org#2497 because the link checker wasn't aware of them. Here, however, we should consider people may be reading the raw `README.md` file in their editor as well. In that case, having reference links is cleaner IMO.

Reply:
As of now I think MD link styles are a personal preference (until we manage to agree on some guidelines, lets discuss elsewhere?) so up to you, esp. in this repo.